scribus/text/boxes.h

```diff
@@ -13,6 +13,7 @@
 #include "frect.h"
 #include "sctextstruct.h"
 #include "scpainter.h"
+#include "storytext.h"
```

Comment:
Hello Khalid
The issue solved

https://github.com/HOST-Oman/scribus/pull/48/files#r56147267

Regards

On Tue, Mar 15, 2016 at 3:15 PM, Khaled Hosny <notifications@github.com>
wrote:

> In scribus/text/boxes.h
> <https://github.com/HOST-Oman/scribus/pull/48#discussion_r56147267>:
>
> > @@ -13,6 +13,7 @@
> >  #include "frect.h"
> >  #include "sctextstruct.h"
> >  #include "scpainter.h"
> > +#include "storytext.h"
>
> This include in not needed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/HOST-Oman/scribus/pull/48/files#r56147267
>